agent-packaging/packaging-scripts/setup_deb.sh

```diff
@@ -13,31 +13,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-source packaging/common.sh 
 
-working_dir=${PWD}
+set -e
 
-sudo apt-get update
+echo "running common script..."
```

Comment:
dont think we need this

Reply:
i think i added this for debugging. i think i added to mark the starting of the `setup_deb.sh`